best-practices.html

```diff
@@ -29,6 +29,7 @@ <h1>Best practices and recommended tools</h1>
                 <li>
                     <a href="#all">For all GitHub users</a>
                     <ul>
+                        <li><a href="#2fa">Enable 2FA</a></li>
```

Comment:
s/Enable 2FA/Enable Two-factor authentication (2FA)/

This is to make it clearer for people that are not familiar with the acronym 2FA.